docs/faq.rst

```diff
@@ -1,23 +1,109 @@
 FAQ
 ===
 
-What happens if I rename my GitHub repository?
-----------------------------------------------
+Contents:
```

Comment:
I don't see this style of TOC at the in any of other pages, although maybe I'm wrong. And it looks a lot different in a local build.

I feel you should either get rid of this section, or we should add it to all our pages, probably the former. For better or for worse, I think we should try to conform to the ReadTheDocs style.

Reply:
Contents doesn't really make sense for most pages, since you know what will be in them from the title. But an FAQ page by definition will have a hodgepodge of questions about a variety of topics (unless there are multiple FAQ pages, each focused on a specific topic, which is also an option). People don't want to have to scroll through the entire page to find out if their question may or may not be answered.